src/apps/filters/geoToH3.c

```diff
@@ -47,9 +47,14 @@ void doCoords(double lat, double lon, int res) {
     GeoCoord g = {.lat = H3_EXPORT(degsToRads)(lat),
                   .lon = H3_EXPORT(degsToRads)(lon)};
 
-    H3Index h = H3_EXPORT(geoToH3)(&g, res);
+    H3Index h;
+    H3Error e = H3_EXPORT(geoToH3)(&g, res, &h);
 
-    h3Println(h);
+    if (e == E_SUCCESS) {
+        h3Println(h);
+    } else {
+        h3Println(H3_NULL);
```

Comment:
Ideally it would print some nice data about which error it ran into to `stderr` instead. :)